cms/static/sass/views/_container.scss

```diff
@@ -239,6 +239,15 @@
             color: $gray-l1;
           }
         }
+
+        .action-copy {
+          width: 100%;
+          border-color: #0075b4;
+
+          &:hover {
+            @extend %btn-primary-blue;
+          }
```

Comment:
This might be a nit, but when I hover over the button, it expands slightly and pushes things down a bit. I think its mainly because the font weight changes with `btn-primary-blue`

Reply:
Yeah I added some padding to the original button to match the hover state.